migrations/hsl/timetables/1676027954830_add_hastus_fields/up.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE vehicle_journey.vehicle_journey
+  ADD COLUMN displayed_label text DEFAULT NULL,
+  ADD COLUMN is_vehicle_type_mandatory boolean NOT NULL DEFAULT false,
+  ADD COLUMN is_backup_journey boolean NOT NULL DEFAULT false,
+  ADD COLUMN is_extra_journey boolean NOT NULL DEFAULT false;
+
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.displayed_label IS 'Displayed name of the journey.';
```

Comment:
I think this should rather be called "displayed_name". For us, "label" are kind of identifiers, so "display_label" sounds a bit weird